### DISCUSSION TITLE
[Idea] Add Product Reference Codes to Wishlist

### DISCUSSION THREAD

**Discussion by mrkalchemy** [PrestaShop/PrestaShop#33575] [Ideas]
### The problem

A lot of B2B customers use reference codes to order goods, as names may not be unique. This can make distinguishing products  on a wishlist difficult and limits their utility in such an environment.

### The solution

Adding the product's reference code to the wishlist display would eliminate this issue. Maybe adding a toggle in the backend for people that prefer a less cluttered appearance would be nice?

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by mrkalchemy** [comment]
I worked out that by altering the product.bundle.js I could add the reference to the products on the wishlist page but I'm sure there must be a better way. That file definitely looks machine generated so I'm guessing there is a template and accompanying script somewhere. If someone can point me in the right direction I could probably do a PR for it.

**Reply by kpodemski** [reply]
Hi! You can find all the sources of the module here: https://github.com/PrestaShop/blockwishlist :)

**Reply by mrkalchemy** [reply]
Thanks, I had already found that but wasn't sure how the templating worked. I think it's using Vue.js but I've never used that before so I was hoping that there was some guide that could help.

**Reply by MatShir** [reply]
I don't recall any guideline, but maybe [on slack](https://www.prestashop-project.org/slack/) you could found some help 🤷
